k8s-charm/lib/charms/juju_dashboard/v0/juju_dashboard.py

```diff
@@ -77,16 +77,19 @@ def __init__(self, data):
                 juju-controller application.
 
         """
+        # Juju controller provides is-juju, while JAAS provides is_juju.
+        is_juju = data.get("is_juju") or data.get("is-juju", True)
```

Comment:
if `data["is_juju"]` is set to "False", this will return True. This might not be an issue, just something to keep in mind